cmd/notation/cert.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/notaryproject/notation-go-lib/signature/x509"
+	"github.com/notaryproject/notation/pkg/config"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	certCommand = &cli.Command{
+		Name:    "certificate",
+		Aliases: []string{"cert"},
+		Usage:   "Manage certificates used for verification",
+		Subcommands: []*cli.Command{
+			certAddCommand,
+			certListCommand,
+			certRemoveCommand,
+			certGenerateTestCommand,
+		},
+	}
+
+	certAddCommand = &cli.Command{
+		Name:      "add",
+		Usage:     "Add certificate to verification list",
+		ArgsUsage: "<path>",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "name",
+				Aliases: []string{"n"},
+				Usage:   "certificate name",
+			},
+		},
+		Action: addCert,
+	}
+
+	certListCommand = &cli.Command{
+		Name:    "list",
+		Usage:   "List certificates used for verification",
+		Aliases: []string{"ls"},
+		Action:  listCerts,
+	}
+
+	certRemoveCommand = &cli.Command{
+		Name:      "remove",
+		Usage:     "Remove certificate from the verification list",
+		Aliases:   []string{"rm"},
+		ArgsUsage: "<name> ...",
+		Action:    removeCerts,
+	}
+
+	certGenerateTestCommand = &cli.Command{
+		Name:      "generate-test",
+		Usage:     "Generates a test RSA key and a corresponding self-signed certificate",
+		ArgsUsage: "<host> ...",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "name",
+				Aliases: []string{"n"},
+				Usage:   "key and certificate name",
+			},
+			&cli.IntFlag{
+				Name:    "bits",
+				Usage:   "RSA key bits",
+				Aliases: []string{"b"},
+				Value:   2048,
+			},
+			&cli.DurationFlag{
+				Name:    "expiry",
+				Aliases: []string{"e"},
+				Usage:   "certificate expiry",
```

Comment:
Can you specify the unit is hour?

Reply:
or days? or leave it for now. Please open an issue to discuss and track.

Reply:
To get this merged, enabling iterative feedback, can we please open an issue for discussion?